README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started
+
+Here's how to get the dev environment setup for our Prysmatic Website:
+1. Via terminal/command line cd into the project's main directory
+2. Run `npm install`
+3. Run `npm start`
+
+## Available Commands
+
+| Command   | Description                                                                                                                                                               |
+|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| npm start | Builds the Prysmatic website and opens it in your default browser                                                                                                         |
+| npm build | Builds the Prysmatic website                                                                                                                                              |
+| npm test  | Sets up a test environment for development (more logging and such)                                                                                                        |
+| npm eject | If you aren’t satisfied with the build tool and configuration choices, you can eject at any time. This command will remove the single build dependency from your project. |
+
+# Become a Shardler
+
+Join our movement to Shardl Ethereum!
+
+At Prysmatic Labs we believe in the ethos of Ethereum. We feel like it's only fitting that Ethereum's future (scaling the network) is built in a way the reflects its core properties of openness and decentralization. At Prysmatic, our team is composed of individuals from around the world who dedicate their free-time and energy to researching and innovating within the realm of **truly decentralized** blockchain scalability.
+
+Scaling a **truly open, and truly decentralized** blockchain is no easy task; especially given the fact that... it's never been done before! But this is what makes it exciting, and this is why we're doing what we're doing.
+
+But we need your help!
+
+Visit our [home page](https://prysmaticlabs.com/) to learn how to become a Shardler.
+
+# Questions/Issues?
+
+Hit us up in our [Gitter Channel](http://gitter.im/prysmaticlabs/geth-sharding?source=orgpage). Our team hangs out almost 24/7 to answer any and all questions relating to scaling Ethereum!
+
+For specific website Issues, just open an Issue in Github!
+
+# Donations
+
+If you have no skills, don't worry, you can still help us out in one way.
+
+Actually, there's two:
+
+1. Purchase some nunchucks [Amazon Nunchucks listings](https://www.amazon.com/s/ref=nb_sb_noss_1?url=search-alias%3Daps&field-keywords=nunchucks), and take a selfie video practicing with Nunchucks, and then post the video in our Gitter channel
```

Comment:
A bit distracting to have this if we want donators to take us seriously IMO

Reply:
Haha I agree this isn't necessary